scripts/patch-realsense-tx2.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+#Break execution on any error received
+set -e
+
+#Locally suppress stderr to avoid raising not relevant messages
+exec 3>&2
+exec 2> /dev/null
+con_dev=$(ls /dev/video* | wc -l)
+exec 2>&3
+
+if [ $con_dev -ne 0 ];
+then
+	echo -e "\e[32m"
+	read -p "Remove all RealSense cameras attached. Hit any key when ready"
+	echo -e "\e[0m"
+fi
+
+#Include usability functions
+source ./scripts/patch-utils.sh
+
+# Get the required tools and headers to build the kernel
+sudo apt-get install linux-headers-generic build-essential git
+
+#Additional packages to build patch
+require_package libusb-1.0-0-dev
+require_package libssl-dev
+
+LINUX_BRANCH=$(uname -r)
+
+kernel_branch=$(choose_kernel_branch $LINUX_BRANCH)
+kernel_name=="kernel-4.4"
```

Comment:
Replace the hard-coded value by parsing the current kernel configuration, e.g. $LINUX_BRANCH above. So when Jetson will switch to another kernel, this script will recognize it.